src/indev/lv_indev.h

```diff
@@ -113,31 +117,81 @@ lv_indev_t * lv_indev_active(void);
  */
 void lv_indev_set_type(lv_indev_t * indev, lv_indev_type_t indev_type);
 
+/**
+ * Set a function pointer to read input device data to the indev
+ * @param indev pointer to an input device
+ * @param read_cb pointer to function pointer to read input device data
```

Comment:
_function pointer_ -> _callback function_

Reply:
Done in 25f295f